frontend/src/charts/ChoroplethMap.tsx

```diff
@@ -208,11 +209,15 @@ export function ChoroplethMap(props: ChoroplethMapProps) {
           type: "shape",
           from: { data: MISSING_DATASET },
           encode: {
-            enter: {
-              tooltip: {
-                signal: missingDataTooltipValue,
-              },
-            },
+            enter:
+              props.hideMissingDataTooltip === undefined ||
```

Comment:
Would it be simpler to check !== true here instead?

Reply:
genius! xD